src/app/containers/RadioSchedule/Canonical/index.jsx

```diff
@@ -4,54 +4,81 @@ import { string } from 'prop-types';
 import styled from 'styled-components';
 import pathOr from 'ramda/src/pathOr';
 import {
+  GEL_GROUP_1_SCREEN_WIDTH_MAX,
+  GEL_GROUP_2_SCREEN_WIDTH_MIN,
   GEL_GROUP_2_SCREEN_WIDTH_MAX,
   GEL_GROUP_3_SCREEN_WIDTH_MIN,
-  GEL_GROUP_3_SCREEN_WIDTH_MAX,
   GEL_GROUP_4_SCREEN_WIDTH_MIN,
 } from '@bbc/gel-foundations/breakpoints';
 import {
+  GEL_SPACING,
   GEL_SPACING_DBL,
   GEL_SPACING_TRPL,
   GEL_SPACING_QUAD,
+  GEL_MARGIN_BELOW_400PX,
+  GEL_MARGIN_ABOVE_400PX,
 } from '@bbc/gel-foundations/spacings';
 import RadioSchedule from '@bbc/psammead-radio-schedule';
 import SectionLabel from '@bbc/psammead-section-label';
 import { Link } from '@bbc/psammead-story-promo';
+import { C_LUNAR } from '@bbc/psammead-styles/colours';
 import { ServiceContext } from '#contexts/ServiceContext';
 import processRadioSchedule from '../utilities/processRadioSchedule';
 import webLogger from '#lib/logger.web';
 
 const logger = webLogger();
 
+const RadioScheduleSection = styled.section.attrs(() => ({
+  role: 'region',
+  'aria-labelledby': 'Radio-Schedule',
+}))`
+  background-color: ${C_LUNAR};
+  padding: 0 ${GEL_MARGIN_BELOW_400PX};
+
+  /* To remove GEL Margins */
+  margin: 0 -${GEL_MARGIN_BELOW_400PX};
+  @media (max-width: ${GEL_GROUP_1_SCREEN_WIDTH_MAX}) {
+    padding-bottom: ${GEL_SPACING_DBL};
+  }
+
+  @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) {
+    padding: 0 ${GEL_MARGIN_ABOVE_400PX};
+    margin: 0 -${GEL_MARGIN_ABOVE_400PX}; /* To remove GEL Margins */
+  }
+`;
+
 const MarginWrapper = styled.div`
-  @media (max-width: ${GEL_GROUP_2_SCREEN_WIDTH_MAX}) {
-    margin-top: ${GEL_SPACING_DBL};
+  margin: 0 auto;
+  padding-bottom: 0;
+  width: 100%; /* Needed for IE11 */
+  @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) {
+    padding-bottom: ${GEL_SPACING_DBL};
   }
   @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
     margin-top: ${GEL_SPACING_TRPL};
-  }
-  @media (max-width: ${GEL_GROUP_3_SCREEN_WIDTH_MAX}) {
-    margin-bottom: ${GEL_SPACING_DBL};
+    padding-bottom: ${GEL_SPACING_DBL};
   }
   @media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) {
+    padding-bottom: ${GEL_SPACING_TRPL};
     margin-bottom: ${GEL_SPACING_TRPL};
+    max-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN};
   }
 `;
 
-const RadioScheduleSection = styled.section.attrs(() => ({
-  role: 'region',
-  'aria-labelledby': 'Radio-Schedule',
-}))`
+const RadioScheduleSectionLabel = styled(SectionLabel)`
   margin: 0 auto;
+  padding-top: 0;
   width: 100%; /* Needed for IE11 */
-  @media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) {
-    max-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN};
+  @media (max-width: ${GEL_GROUP_2_SCREEN_WIDTH_MAX}) {
+    padding-top: ${GEL_SPACING};
```

Comment:
Should we be adding padding here or adjusting the margin-top of section label based on the breakpoints?

Reply:
Added the padding to increase the `C_LUNAR` background height above the SectionLabel.